pipcompilemulti/cli_v1.py

```diff
@@ -33,11 +33,19 @@
 @click.option('--only-name', '-n', multiple=True,
               help='Compile only for passed environment names and their '
                    'references. Can be supplied multiple times.')
-@click.option('--upgrade/--no-upgrade', default=True,
-              help='Upgrade package version (default true)')
+@click.option('--upgrade/--no-upgrade',
+              help='Upgrade package version (default False)')
```

Comment:
I agree, that `--no-upgrade` is consistent with `pip-compile`. But I don't want to change the default behavior as it will break existing usage of the tool.
There is (experimental) CLI v2, that fixes this problem, but it's out of scope of this PR I guess.
I think, it's okay if `--upgrade-package` silently overrides `--upgrade` (switching it to `--no-upgrade`).

Reply:
I have made --upgrade-package force --no-upgrade.